build/secondary/third_party/libwebp/BUILD.gn

```diff
@@ -14,47 +17,43 @@ config("libwebp_defines") {
     # This makes WebP decode to BGR_565 when we ask for RGB_565.
     # (It also swaps the color order for 4444, but we don't care today.)
     "WEBP_SWAP_16BIT_CSP",
-    "WEBP_EXTERN(x)= extern x",
```

Comment:
I'm pretty sure this is what messed up the symbol exposure. I took it out because it wasn't in the Skia build rules but I'm almost certain we put it here to make sure the symbols don't get exposed.